tup-cms/settings_custom.py

```diff
@@ -1,57 +1,31 @@
-# TACC CMS Per-Site Resources - Configuration
-
-# In a project-specific configuration file like this, edit existing configuration values from `cms-site-template:/taccsite_cms/default_secrets.py` that must change to suit this project.
-
-# Until this custom configuration is automatically applied, to use it one must copy the content of it and append it to `cms-site-template:/taccsite_cms/secrets.py`
-
-# TUP CONFIGURATION
-
-########################
-# DJANGO SETTINGS
-########################
-
-_LDAP_ENABLED = False
+# CUSTOM SETTINGS VALUES.
+# TACC WMA CMS SITE:
+# *.TUP.TACC.UTEXAS.EDU
 
 ########################
 # DJANGO CMS SETTINGS
 ########################
 
-_CMS_TEMPLATES = (
-    ('tup-cms/templates/fullwidth.html', 'Fullwidth')
+CMS_TEMPLATES = (
+    ('tup-cms/templates/fullwidth.html', 'Fullwidth'),
+    ('guide.html', 'Guide'),
+    ('guides/getting_started.html', 'Guide: Getting Started'),
+    ('guides/data_transfer.html', 'Guide: Data Transfer'),
+    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
+    ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
 )
```

Comment:
The guides templates are probably not needed.
- guides were always a short term fix
- only intended for portal
- should not be unnecessary when we have a main TACC docs

I suggest either:

- [your initial plan to have them, but commented](https://github.com/TACC/Core-CMS-Resources/pull/74/commits/12c6d24)

	```suggestion
	CMS_TEMPLATES = (
 		('fullwidth.html', 'Full Width'),
 		('tup-cms/templates/fullwidth.html', 'Full Width'),
 
	    # ('guide.html', 'Guide'),
	    # ('guides/getting_started.html', 'Guide: Getting Started'),
	    # ('guides/data_transfer.html', 'Guide: Data Transfer'),
	    # ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
	    # ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
	)
	```

- or to remove them

	```suggestion
	CMS_TEMPLATES = (
 		('fullwidth.html', 'Full Width'),
 		('tup-cms/templates/fullwidth.html', 'Full Width'),
	)
	```

<details>
<summary>Note to Self</summary>

In https://github.com/TACC/Core-CMS-Resources/pull/78/files, the `tup-cms` template will be removed. Doing so here may cause confusion or unnecessary delay.

</details>